### ISSUE TITLE
JAWS reads Contextual Menu Button as "Button, Menu" which is incorrect

### ISSUE THREAD

**Issue by drotara** [microsoft/fluentui#17186]
### Environment Information

Package version(s): 7.153.2 +
Browser and OS versions: Edge 88.0.705.50

### Describe the issue:
JAWS when encountering the contextual menu button reads it as Button, menu which is confusing. It should be reading it as "button, has popup collapsed" so that the user knows there is additional information but its collapsed and to select the button to expand. 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/adrotar/pen/eYBVpWm

#### Actual behavior:
JAWS reads it as "Button, Menu" 

#### Expected behavior:
Should read as "Button, has popup collapsed" 

### Documentation describing expected behavior
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7ba12f3779-bdf1-40c2-bbcc-0e314ea94858%7d&cid=4eda0f34-0149-402f-998f-71fc7732f2a7

**Comment by gouttierre** [comment]
@drotara - thanks for filing this issue. I have checked and confirmed your repro - the citation goes along the lines of ` new item new item button menu expanded` and for the collapse it does not cite the state change.

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue? thanks 
cc @smhigley

**Comment by drotara** [comment]
So has this issue actually been fixed or what happened?

**Comment by khmakoto** [comment]
@drotara I've reopened the issue.
